tasks/docker-mounts.yml

```diff
@@ -0,0 +1,17 @@
+---
+platform: linux
+
+image_resource:
+  type: registry-image
+  source: {repository: concourse/unit}
+
+params:
+  KUBE_CONFIG:
+  RELEASE_NAME:
```

Comment:
Are these params used anywhere? Noticed they're not used in the new script and can't see how k8s-related envs could be useful here. Guessing this is leftover from a copy-paste?

Reply:
Yup, good catch!